### ISSUE TITLE
Error - Call to undefined method getAttribute

### ISSUE THREAD

**Issue by bwmarkle** [BoldGrid/boldgrid-inspirations#20]
![image](https://user-images.githubusercontent.com/5925563/57780616-9710be00-76f6-11e9-9c54-54c25c78b88b.png)

**Comment by bwmarkle** [comment]
All plugins are up to date. Things work on php 5.6, but not 7.3

**Comment by bwmarkle** [comment]
### wp-core ###
version: 5.2
site_language: en_US
user_language: en_US
permalink: /%postname%/
https_status: false
user_registration: 0
default_comment_status: open
multisite: false
user_count: 2
dotorg_communication: true
### wp-paths-sizes ###
wordpress_path: /home/user/public_html
wordpress_size: 40.74 MB (42722290 bytes)
uploads_path: /home/user/public_html/wp-content/uploads
uploads_size: 452.10 MB (474061297 bytes)
themes_path: /home/user/public_html/wp-content/themes
themes_size: 16.50 MB (17305038 bytes)
plugins_path: /home/user/public_html/wp-content/plugins
plugins_size: 33.18 MB (34793684 bytes)
database_size: 6.10 MB (6393388 bytes)
total_size: 548.63 MB (575275697 bytes)
### wp-active-theme ###
name: Resolve
version: 1.19.3
author: the BoldGrid team
author_website: https://www.boldgrid.com/the-team
parent_theme: none
theme_features: editor-style, automatic-feed-links, title-tag, html5, post-thumbnails, custom-background, infinite-scroll, woocommerce, wc-product-gallery-zoom, wc-product-gallery-lightbox, wc-product-gallery-slider, menus, widgets
theme_path: /home/user/public_html/wp-content/themes/boldgrid-resolve
### wp-themes (2) ###
Twenty Fourteen: version: 2.7, author: the WordPress team
Twenty Nineteen: version: 1.4, author: the WordPress team
### wp-plugins-active (8) ###
Akismet Anti-Spam: version: 4.1.2, author: Automattic
BoldGrid Backup: version: 1.9.3, author: BoldGrid
BoldGrid Easy SEO: version: 1.6.2, author: BoldGrid <support@boldgrid.com>
BoldGrid Gallery: version: 1.5, author: BoldGrid.com <wpb@boldgrid.com>
BoldGrid Inspirations: version: 2.0.2, author: BoldGrid <support@boldgrid.com>
BoldGrid Ninja Forms: version: 1.5, author: BoldGrid.com
BoldGrid Staging: version: 1.5.1, author: BoldGrid.com <wpb@boldgrid.com>
Post and Page Builder: version: 1.10.1, author: BoldGrid <support@boldgrid.com>
### wp-media ###
image_editor: WP_Image_Editor_GD
imagick_module_version: Not available
imagemagick_version: Not available
gd_version: bundled (2.1.0 compatible)
ghostscript_version: 8.70
### wp-server ###
server_architecture: Linux 2.6.32-042stab127.2 x86_64
httpd_software: Apache
php_version: 5.6.40 64bit
php_sapi: cgi-fcgi
max_input_variables: 1000
time_limit: 30
memory_limit: 256M
max_input_time: 60
upload_max_size: 2M
php_post_max_size: 8M
curl_version: 7.64.0 OpenSSL/1.0.2r
suhosin: false
imagick_availability: false
htaccess_extra_rules: true
### wp-database ###
extension: mysqli
server_version: 5.6.43
client_version: mysqlnd 5.0.11-dev - 20120503 - $Id: 76b08b24596e12d4553bd41fc93cccd5bac2fe7a $
### wp-constants ###
WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /home/user/public_html/wp-content
WP_PLUGIN_DIR: /home/user/public_html/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
### wp-filesystem ###
wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

**Comment by cssjoe** [comment]
Variable defined at: https://github.com/BoldGrid/boldgrid-inspirations/blob/master/includes/class-boldgrid-inspirations-my-inspiration.php#L302
Method getAttribute used at: https://github.com/BoldGrid/boldgrid-inspirations/blob/master/includes/class-boldgrid-inspirations-my-inspiration.php#L329

The method was introduced in the library v2.8.0 in commit: https://github.com/BoldGrid/library/commit/ef1a462aad7ab70a0ee32c736e4f7ac79f6cb727
Check the library version registered and loaded.

**Comment by bwmarkle** [comment]
The issue is no longer happening, but I tested the library versions and found this:

Inspirations NOT active
```
post-and-page-builder/post-and-page-builder.php		2.8.0.0
boldgrid-backup/boldgrid-backup.php			2.7.7.0
```

Inspirations IS active
```
post-and-page-builder/post-and-page-builder.php		2.8.0.0
boldgrid-backup/boldgrid-backup.php			2.7.7.0
boldgrid-inspirations/boldgrid-inspirations.php		2.8.0.0
```

**Comment by bwmarkle** [comment]
![image](https://user-images.githubusercontent.com/5925563/57783680-8c592780-76fc-11e9-945b-8bf395ba2f62.png)

**Comment by bwmarkle** [comment]
https://github.com/BoldGrid/boldgrid-inspirations/commit/c9bdf69c7d625fce7c3abefa8becbb238e3ca239 fixes #20
